requirements/make_base.py

```diff
@@ -0,0 +1,44 @@
+import sys
+from argparse import ArgumentParser
+
+import tomli
+
+parser = ArgumentParser()
+parser.add_argument(
+    "--nightly",
+    default="",
+    help="List of dependencies to install from main branch for nightly tests, "
+    "separated by commas.",
+)
+args = parser.parse_args()
+
+header = "# Generated by 'tox -e deps', DO NOT EDIT MANUALLY!'\n"
+
+with open("../pyproject.toml", "rb") as toml_file:
+    pyproject = tomli.load(toml_file)
+    dependencies = pyproject["project"].get("dependencies")
+    if not dependencies:
+        raise RuntimeError("No dependencies found in pyproject.toml")
+    dependencies = [dep.strip().strip('"') for dep in dependencies]
+
+with open("base.in", "w") as f:
+    f.write(header)
+    f.write("\n".join(dependencies))
+
+
+def as_nightly(name: str) -> str:
+    if name == "scipp":
+        version = f"cp{sys.version_info.major}{sys.version_info.minor}"
+        base = "https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly"
+        suffix = "manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
```

Comment:
How are we making sure the `2_17` and `2014` numbers in the suffix are correct? I can see they come from the copier template, but they are hardcoded there as well. Do they depend on the version of `cibuildwheel` we are using to build scipp?

Reply:
> Do they depend on the version of cibuildwheel we are using to build scipp?

Yes, essentially. We have never changed this since the beginning of our wheel builds, so it will be a really rare thing. I do not want to remove it, since we may later also want to add mac, windows, ...

Reply:
Well, they might actually have changed, but I think the manylinux2014 was already there when we started, and the 2_17 is even older, iirc.